Cargo.lock

```diff
@@ -0,0 +1,1161 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I don't think committing the lockfile makes sense without having the corresponding stable test CI job

Reply:
Yeah, it doesn't need to be here.